tools/rdock/rdock.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>

```

Comment:
We have it already installed and available as a module but it would be good to have conda recipe for others